content_scripts/CameraFollowCoordinator.js

```diff
@@ -90,35 +90,29 @@ export class CameraFollowCoordinator {
         if (this.currentFollowTarget.followable) {
             this.currentFollowTarget.followable.onCameraStartedFollowing();
         }
-
-        // if the followable specifies a frameKey, try to focus on that envelope when following
-        if (typeof this.currentFollowTarget.followable.frameKey !== 'undefined') {
-            realityEditor.envelopeManager.focusEnvelope(this.currentFollowTarget.followable.frameKey );
-        }
-
+        this.followDistance = followDistance;
```

Comment:
I think it might be good to wrap this line in a `if (typeof followDistance !== 'undefined')` to guard against the case that someone accidentally calls the `follow` function without the distance parameter – in that case it would preserve the current follow distance, instead of setting it to undefined.